FileIO/XmlIO/Boost/BoostXmlGmlInterface.cpp

```diff
@@ -328,17 +279,13 @@ bool BoostXmlGmlInterface::write()
 	addPolylinesToPropertyTree(geometry_set);
 	addSurfacesToPropertyTree(geometry_set);
 
-#if BOOST_VERSION <= 105500
-	boost::property_tree::xml_writer_settings<char> settings('\t', 1);
-#else
 	boost::property_tree::xml_writer_settings<std::string> settings('\t', 1);
-#endif  // BOOST_VERSION
 	write_xml(_out, pt, settings);
 	return true;
 }
 
 void BoostXmlGmlInterface::addSurfacesToPropertyTree(
-	BaseLib::ConfigTree & geometry_set)
+	boost::property_tree::ptree & geometry_set)
```

Comment:
The ptree is used here because the caller of `addSurfaces..` does not understand `ConfigTreeNew`, isn't it? Or maybe I plain wrong...